AmazonChimeSDKDemo/AmazonChimeSDKDemo/utils/JoinRequestService.swift

```diff
@@ -20,8 +20,10 @@ class JoinRequestService: NSObject {
 
     static func postJoinRequest(meetingId: String,
                                 name: String,
+                                overrideUrl: String,
                                 completion: @escaping (MeetingSessionConfiguration?) -> Void) {
-        let url = AppConfiguration.url.hasSuffix("/") ? AppConfiguration.url : "\(AppConfiguration.url)/"
+        let url = overrideUrl.count > 0 ? overrideUrl : (AppConfiguration.url.hasSuffix("/") ? AppConfiguration.url : "\(AppConfiguration.url)/")
```

Comment:
use isEmpty instead of count >0
We probably want to keep the same logic of adding "/" if QA does not add one when overriding

Reply:
Done.

Reply:
done